Source/Core/Runtime/Configuration/BaseRuntimeConfiguration.cs

```diff
@@ -143,8 +150,44 @@ public virtual async Task<IProcess> LoadProcess(string path)
                     throw new ArgumentException("Given path is null or empty!");
                 }
 
-                byte[] serialized = await FileManager.Read(path);
-                return Serializer.ProcessFromByteArray(serialized);
+                int index = path.LastIndexOf("/");
+                string processFolder = path.Substring(0, index);
+                string manifestPath = $"{processFolder}/{ManifestFileName}.{Serializer.FileFormat}";
+
+                IProcessAssetManifest manifest;
+
+                if (await FileManager.Exists(manifestPath))
+                {
+                    byte[] manifestData = await FileManager.Read(manifestPath);
+                    manifest = Serializer.ManifestFromByteArray(manifestData);
+                }
+                else
+                {
+                    Debug.LogError("Manifest not found.");
+                    return null;
+                }
+
+                IProcessAssetStrategy assetStrategy = ReflectionUtils.CreateInstanceOfType(ReflectionUtils.GetConcreteImplementationsOf<IProcessAssetStrategy>().FirstOrDefault(type => type.FullName == manifest.AssetStrategyTypeName)) as IProcessAssetStrategy;
+
+                string processAssetPath = $"{processFolder}/{manifest.ProcessFileName}.{Serializer.FileFormat}";
```

Comment:
Creating a sting "folder + file name + file format" is a reoccurring thing in multipe places how do you think about a helper method like GetlFilePahtName(string folder, string fileName, string fileFormat).
Do we have a place for such helpers?

Reply:
Only advantage would be if for some reason the string setup changes we will not need to change it every where.
Ok might be not worth it.